src/Umbraco.Web.UI.Client/src/views/common/overlays/querybuilder/querybuilder.controller.js

```diff
@@ -154,7 +165,13 @@
                 });
 
         }, 200);
-        
+
+        function datePickerChange(event) {
+            templateQueryResource.postTemplateQuery(vm.query)
```

Comment:
Correct me if I am wrong @madsrasmussen but the main directive model is `filter.constraintValue` and I was expecting this function to update `vm.query` to include the chosen date to post to the server for the query to be made.